lib/perl/Genome/Model/Tools/Picard/CalculateHsMetrics.pm

```diff
@@ -47,8 +47,8 @@ class Genome::Model::Tools::Picard::CalculateHsMetrics {
             picard_param_name => 'BAIT_SET_NAME',
         },
         metric_accumulation_level => {
-            is => 'String',
-            doc => 'The level(s) at which to accumulate metrics. Possible values: {ALL_READS, SAMPLE, LIBRARY, READ_GROUP} This option may be specified 0 or more times. This option can be set to \'null\' to clear the default list.',
+            is => 'Text',
+            is_many => 1,
```

Comment:
It isn't set on any of the tools for this parameter but I think it's a good idea. I will add it.